mirage/factories/utils.ts

```diff
@@ -114,3 +141,39 @@ export function createRegistrationMetadata(
     }
     return registrationMetadata;
 }
+
+/**
+ * Create registrationResponse according to schemaBlocks
+ */
+export function createRegistrationResponses(
+    registrationSchema: ModelInstance<MirageRegistrationSchema>,
+) {
+    const schemaBlocks = registrationSchema.schemaBlocks!;
+    const schemaBlockGroups = getSchemaBlockGroups(schemaBlocks);
+    const registrationResponses = {} as RegistrationResponses;
+    for (const group of schemaBlockGroups) {
+        switch (group.groupType) {
+        default:
+            break;
+        case 'long-text-input':
+            registrationResponses[group.registrationResponseKey!] = faker.lorem.paragraph();
```

Comment:
If `group.registrationResponseKey` could be `undefined`, let's wrap this whole switch in a condition to test that it's not.

Reply:
Also, would be cleaner to pull `registrationResponseKey` out:
```ts
const { groupType, registrationResponseKey } = group;
if (registrationResponseKey) {
    switch(groupType) {
    case 'foo':
        registrationResponses[registrationResponseKey] = ....
```